appserver/tests/payara-samples/samples/client-certificate-validator/src/main/java/fish.payara.samples.security/HelloServlet.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
This file seems to be have been refactored weird.
Check the file path - rather than `src/main/java/fish/payara/samples/security` it's `src/main/java/fish.payara.samples.security`